### ISSUE TITLE
stratum_bmv2 advertises only one port even if chassis_config defines many

### ISSUE THREAD

**Issue by ccascone** [stratum/stratum#438]
**Describe the bug**
When using ONOS with stratum_bmv2, ONOS discovers only one port even if the chassis_config defines more than one port. 

**Environment**
Running the latest `mn-stratum` image.

```
docker image inspect opennetworking/mn-stratum:latest@sha256:093202ac7a679ed370396ecae0f48f6883bfc371773afcdfb591fdf55339131a | grep vcs
                "LABEL vcs-ref=651a51b2610418f143129409c56d71751ebab172"
                "vcs-ref": "651a51b2610418f143129409c56d71751ebab172"
                "vcs-ref": "651a51b2610418f143129409c56d71751ebab172"
```

**To Reproduce**
Follow steps in `tools/mininet/examples/trellis` to run a 2x2 fabric controlled by ONOS.
After pushing the netcfg, check ports discovered using ONOS CLI:
```
onos@root > ports -s                                                                                                                                                                                                                                                                                            02:22:04
id=device:leaf1, available=true, role=MASTER, type=SWITCH, driver=stratum-bmv2:org.onosproject.pipelines.fabric
  port=[leaf1-eth4](0), state=enabled, type=copper, speed=10000 , last-changed=1605752509252098460
id=device:leaf2, available=true, role=MASTER, type=SWITCH, driver=stratum-bmv2:org.onosproject.pipelines.fabric
  port=[leaf2-eth2](0), state=enabled, type=copper, speed=10000 , last-changed=1605752508982148209
id=device:spine1, available=true, role=MASTER, type=SWITCH, driver=stratum-bmv2:org.onosproject.pipelines.fabric
  port=[spine1-eth1](0), state=enabled, type=copper, speed=10000 , last-changed=1605752509401583571
id=device:spine2, available=true, role=MASTER, type=SWITCH, driver=stratum-bmv2:org.onosproject.pipelines.fabric
  port=[spine2-eth2](0), state=enabled, type=copper, speed=10000 , last-changed=1605752508814044738
```

**Expected behavior**
ONOS should discover more than one port for each switch.

**Additional context**
Chassis_config and log from leaf1:
[chassis-config.txt](https://github.com/stratum/stratum/files/5563950/chassis-config.txt)
[stratum_bmv2.log](https://github.com/stratum/stratum/files/5563953/stratum_bmv2.log)

**Comment by pudelkoM** [comment]
This line indicates a failed SDN port lookup: `E20201119 02:21:47.990556   209 bmv2_switch.cc:186] StratumErrorSpace::ERR_UNIMPLEMENTED: DataRequest field hardware_port is not supported yet: hardware_port { node_id: 1 port_id: 1 }.`

https://github.com/stratum/stratum/blob/651a51b2610418f143129409c56d71751ebab172/stratum%2Fhal%2Flib%2Fbmv2%2Fbmv2_chassis_manager.cc#L278 is missing a `break` causing a fall through to `RETURN_ERROR(ERR_INTERNAL) << "Not supported yet";`.

**Comment by pudelkoM** [comment]
Confirmed resolved by #439.
